react/features/contact-list/components/ContactListPanel.web.js

```diff
@@ -67,16 +67,35 @@ class ContactListPanel extends Component {
      * @returns {ReactElement[]}
      */
     _renderContacts() {
+        const { t } = this.props;
+        const meString = t('me');
+
         return this.props._participants.map(participant => {
-            const { id, name } = participant;
+            const { id, local, name } = participant;
+            let displayName;
+
+            // FIXME this duplicates the logic from SpeakerStats.js, but
+            // currently it seems that the
+            // interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME is freely used in
+            // multiple other places. A proper fix should take care of all
+            // usages. Also conference.js has getParticipantDisplayName which
+            // generates HTML span making things even harder to be cleaned up
+            // properly.
+            if (local) {
+                displayName
+                    = name ? `${name} (${meString})` : meString;
+            } else {
+                displayName
+                    = name ? name : interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME;
+            }
```

Comment:
I think trying to remove the duplication between SpeakerStats and ContactList would help any feeling of uneasiness you may have. This style of name display with "(me)" might also be used in the local video display too. I think all three of those are under react and would have access to translate at the time of render.